pkg/apis/pingcap/v1alpha1/tidbclusterautoscaler_types.go

```diff
@@ -85,6 +85,7 @@ type TidbClusterAutoScalerSpec struct {
 type TikvAutoScalerSpec struct {
 	BasicAutoScalerSpec `json:",inline"`
 
+	// Deprecated in v1.1.3, planned for removal in v1.3
```

Comment:
Why remove this?
It's not reasonable to scale as soon as the metric exceeds the threshold.
BTW, even if the PD will handle the metrics later, we can remove this when the similar function is implemented in PD. WDYT @cofyc

Reply:
I believe this is in @Yisaer's plan, replaced by PD before v1.3.

Reply:
I think the current de-noise design in operator is too complicated and haven't been verified whether it is necessary. If we decided let PD make the auto-scaling plan, I think operator could only expose api and do the execution. That will be quite simple and clear, easy to know the status and debug.